exercises/linked-list/dune

```diff
@@ -0,0 +1,16 @@
+(executable
+ (name test)
+ (libraries base oUnit))
+
+(alias
+  (name    runtest)
+  (deps    (:x test.exe))
+  (action  (run %{x})))
+
+(alias
+  (name    buildtest)
+  (deps    (:x test.exe)))
+
+(env
+  (dev
+    (flags (:standard -warn-error -A))))
```

Comment:
I notice that most (but not all of) the files you generate don't have line termination and the files generated by `configlet` do.

I don't have my development computer at hand, so I can't say if we are consistent about this across the track.

The easiest consistency we could aim for is across single exercises, which I think we should aim for either way.

Since README.md is forced to have line endings, I think this and other files should have, too.

Feel free to object and argue otherwise.